src/roi/__tests__/waterShed.test.ts

```diff
@@ -168,16 +167,16 @@ describe('Test WaterShed Roi generation', () => {
     const mask = image.threshold({ algorithm: 'otsu' });
```

Comment:
We should start from a mask and not create it from an image, because we are not testing otsu in this test, only watershed